en/docs/services/create-your-first-service.md

```diff
@@ -1,177 +1,250 @@
+        var totalCasesPerMillion = totalCases / population;
+        json payload = {TotalCasesPerMillion: totalCasesPerMillion};
+        check caller->respond(payload);
+    }
+}
 
-    
-To validate the service, click **Run & Test**. The following is logged to indicate that you have successfully started the service.
+```
 
-![Service Started Log](../assets/img/services/service-started-notification.png){.cInlineImage-half}
-    
-Now you are ready to test the service.
-   
-## Step 3: Test the service
+## Step 6: Test the service
+
+To test the `CovidStatus` service you created, follow the procedure below:
 
-To test the `covid-stats` service, follow this procedure:
+1. Click **Run & Test**.
 
-1. Click the **Test** icon in the left pane.
+    The following logs appear to indicate that the service has successfully started.
+    
+    ```
+    Starting application...
+    [ballerina/http] started HTTP/WS listener 0.0.0.0:8090
+    ```
+   
+2. Click the **Test** icon in the left pane.
 
     ![Test Icon](../assets/img/services/test-icon.png){.cInlineImage-bordered}
 
-2. In the section that opens to the right of the page, click **GET**.
+3. In the test view that opens to the right of the page, click **GET**.
 
-3. Click **Try it out**, and then click **Execute**.
+4. Click **Try it out**, and then enter the following information in the fields that appear.
 
-The following figure shows the server response to the search results for the United States COVID-19 statistics:
+    | **Field**     | **Value**                                         |
+    |---------------|---------------------------------------------------|
+    | **country**   | `USA`                                            |
+    | **date**      | `2019`                                         |
+   
+5. Click **Execute**.
+
+    The response is displayed as follows:
 
-![Server Response](../assets/img/services/server-response.png){.cInlineImage-half}
+    ![Response in the test view](../assets/img/services/test-view-response.png){.cInlineImage-bordered}
 
-Now that you have verified that the `covid-stats` service works as expected, you can deploy it.
 
-## Step 4: Deploy the service
+## Step 7: Deploy the service
 
-To deploy the `covid-stats` service, follow this procedure:
+To deploy the `CovidStatus` service, follow the procedure below:
 
 1. Click the **Go Live** icon in the left pane.
 
-    ![Test Icon](../assets/img/services/deploy-icon.png){.cInlineImage-bordered}
+    ![Deploy icon](../assets/img/services/deploy-icon.png){.cInlineImage-bordered}
 
 2. To deploy the service, click **Deploy**.
 
-    The status of the service changes to **Deployed**, and the following logs show that the service is successfully deployed:    
+    The status of the service changes to **Deployed**, and the following logs appear to indicate that the product is successfully deployed.    
 
     ![Deployment logs](../assets/img/services/deployment-logs.png){.cInlineImage-full}
```

Comment:
This image is incorrect. It shows `Run & Test` Tab. Instead it should show the `Go Live` tab

Reply:
Same applies for the next step (Step 3 ) image as well.

Reply:
Fixed